dbms/src/Core/Spiller.cpp

```diff
@@ -260,8 +258,16 @@ BlockInputStreams Spiller::restoreBlocks(UInt64 partition_id, UInt64 max_stream_
         /// clear the spilled_files so we can safely assume that the element in spilled_files is always not nullptr
         partition_spilled_files.clear();
     }
+    if (all_constant_block_rows[partition_id] > 0)
```

Comment:
Use `if else` to skip restore from file for a spiller that only spill all constant block?

Reply:
ok, update the code like
```
if (is_all_constant)
{
    xxxxx
}
else
{
    the origin codes.
}
```